src/common/charts/shared.js

```diff
@@ -0,0 +1,79 @@
+export const chartFontFamily = '"Market Sans", Arial, sans-serif',
```

Comment:
Still not great to have this defined statically here instead of referencing something global, but we can change this in the future.